tasks/Task.cpp

```diff
@@ -475,26 +476,10 @@ void Task::transmissionConfiguration(Arena::IDevice& device)
         "StreamBufferHandlingMode",
         "NewestOnly");
 
-    // Use max supported packet size. We use transfer control to ensure that only one
-    // camera is transmitting at a time.
-    LOG_INFO_S << "Setting StreamAutoNegotiatePacketSize.";
-    Arena::SetNodeValue<bool>(device.GetTLStreamNodeMap(),
-        "StreamAutoNegotiatePacketSize",
-        true);
-
-    GenApi::CIntegerPtr mtu =
-        device.GetNodeMap()->GetNode("DeviceStreamChannelPacketSize");
-
-    if (config.mtu_threshold > mtu->GetValue()) {
-        throw runtime_error("MTU negotiated is less than expected! Expected: " +
-                            to_string(config.mtu_threshold) +
-                            " current: " + to_string(mtu->GetValue()));
-    }
-
     LOG_INFO_S << "Setting StreamPacketResendEnable.";
     Arena::SetNodeValue<bool>(device.GetTLStreamNodeMap(),
         "StreamPacketResendEnable",
-        true);
+        config.stream_packet_resend);
```

Comment:
*This* is why you don't mix things. I can't event cherry-pick that commit since you put it inside another unrelated one.